### ISSUE TITLE
Confirmation of Votes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halalaninitiative/halalan#46]
```
What steps will reproduce the problem?
1. People will voter their favorite candidates
2. People will click the submit ballot link
3. Some people will not notice that they still need to confirm their
ballots and they logout

Please provide any additional information below.
Suggestions:
#1 Place a link both at the top and the bottom
(http://groups.google.com/group/usapan/browse_thread/thread/6c9a53f76cdf03c4/269
59c1b8dd0a2e1)
#2 A pop-message could also be placed when the voter arrives at the confirm
page, thus focusing his/her attention to a particular "out-of place"
object. The message could be something like please check/review your votes
before proceeding/clicking the confirm button/confirming.

```

Original issue reported on code.google.com by `adobos...@gmail.com` on 6 Mar 2008 at 1:03

**Comment by GoogleCodeExporter** [comment]
```
Reminders and warnings were already added at the top of the concerned pages. 
Please
checkout the latest revision and tell me what you think.
```

Original comment by `djclue...@gmail.com` on 8 Mar 2008 at 4:15
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**, **Priority-High**, **Component-UI**, **Usability**
- Removed labels: **Type-Defect**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Changed status to Fixed since this issue has already been addressed in 1.3.0.
```

Original comment by `waldemar...@gmail.com` on 2 Jul 2008 at 3:51
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `aicapa...@gmail.com` on 19 Jun 2010 at 2:10
- Changed state: **Verified**
